molr-mole-core/src/main/java/io/molr/mole/core/runnable/lang/ExecutionStrategyConfiguration.java

```diff
@@ -0,0 +1,43 @@
+package io.molr.mole.core.runnable.lang;
+
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import com.google.common.collect.Sets;
+
+import io.molr.commons.domain.ExecutionStrategy;
+
+public class ExecutionStrategyConfiguration {
+	
+	private ExecutionStrategy defaultStrategy = ExecutionStrategy.PAUSE_ON_ERROR;
+	private Set<ExecutionStrategy> allowedStrategies = Sets.immutableEnumSet(ExecutionStrategy.PAUSE_ON_ERROR);
+	
+	/**
+	 * Sets the default execution strategy for this mission.
+	 * @param executionStrategy
+	 * @return
+	 */
+	public ExecutionStrategyConfiguration defaultsTo(ExecutionStrategy executionStrategy) {
```

Comment:
Consider separating the DSL from the builder.
In the DSL it makes sense to e.g. have 

`defaultsTo(..).allowed(...)`

However, e.g. `enforceTo(...).allowed(...)` makes no sense...

Therefore, I think haveing dedicated objects for the DSL which pass along a builder would be the better match.